frontend/src/events.js

```diff
@@ -109,8 +109,18 @@ const events = {
 
     let options = {type, seats, title, isPrivate, modernOnly, totalChaos};
 
+    if (gamesubtype === "decadent") {
```

Comment:
Instead of using the "App.state.setsDraft" I would use a specific property to decouple between the regular mode and the decadent one.

Reply:
Good suggestion. I'll add `App.state.setsDecadentDraft` or something similar.

Reply:
https://github.com/dr4fters/dr4ft/pull/990/commits/e6fd5e583f47f9f5ecdc236fb94d156992346acb